packages/ckeditor5-language/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@ckeditor/ckeditor5-language",
+  "version": "25.0.0",
+  "description": "Block quote feature for CKEditor 5.",
+  "keywords": [
+    "ckeditor",
+    "ckeditor5",
+    "ckeditor 5",
+    "ckeditor5-feature",
+    "ckeditor5-plugin"
+  ],
+  "main": "src/index.js",
+  "dependencies": {
+    "ckeditor5": "^25.0.0",
+    "@ckeditor/ckeditor5-utils": "^25.0.0"
```

Comment:
It's not a correct dependency. You can only import things from `ckeditor5` due to DLLs. Can you run the dependency check task to see if it catches this (it should)?

Reply:
Do you mean `yarn run check-dependencies` script? If yes, it's not returning any error 🤔 

![image](https://user-images.githubusercontent.com/10279586/109306601-ae735980-783f-11eb-9e16-c57d705fd8a4.png)

Reply:
On the other hand, when I changed it into `ckeditor5/src/utils` linter says it's incorrect and asks for `@ckeditor/ckeditor5-utils`: 
![image](https://user-images.githubusercontent.com/10279586/109308069-d368cc00-7841-11eb-8478-714fdf15fd62.png)

Reply:
We've talked about it with @niegowski and he explained to me that the import in `language/src/utils.js` file was invalid in the first place, hence this linter message.